scripts/hcp_preproc_and_analysis.py

```diff
@@ -349,8 +350,8 @@ def run_suject_level1_glm(subject_data,
 
     # fit GLM
     def tortoise(*args):
-        print args
-        print (
+        print(args)
+        print(()
```

Comment:
Yeah this is my automatic sub-standard regex replacement. Oh well I should have used futurize or something. I'll look closer at this.